src/MailCollector.php

```diff
@@ -2249,13 +2249,15 @@ private function getMessageIdExtractPattern(): string
        // old format with related item:     GLPI-{$itemtype}-{$items_id}.{$time}.{$rand}@{$uname}
        // new format without related item:  GLPI_{$uuid}.{$time}.{$rand}@{$uname}
        // new format with related item:     GLPI_{$uuid}-{$itemtype}-{$items_id}.{$time}.{$rand}@{$uname}
+       // new format with related item:     GLPI_{$uuid}-{$itemtype}-{$items_id}.{$time}.{$rand}/{eventname}@{$uname}
 
         return '/GLPI'
          . '(_(?<uuid>[a-z0-9]+))?' // uuid was not be present in old format
          . '(-(?<itemtype>[a-z]+))?' // itemtype is not present if notification is not related to any object and was not present in old format
          . '(-(?<items_id>[0-9]+))?' // items_id is not present if notification is not related to any object
-         . '\.[0-9]+' // time()
-         . '\.[0-9]+' // rand()
+         . '(\.[0-9]+)?' // time()
+         . '(\.[0-9]+)?' // rand()
+         . '(\/\w+)?' // eventname
```

Comment:
This is not related to headers modified in this PR, so there is nothing to change here. Indeed, this is related to ID generated by `NotificationTarget::getMessageID()`

Reply:
This PR modifies the format of the header ```References```  and ```In-Reply-To```.
So from ```MailCollector.php``` it is necessary to modify the way GLPI will try to know if it is a noitifcation from itself
IMHO

Reply:
ok, I understand now